dashboard/app/dsl/base_dsl.rb

```diff
@@ -20,8 +20,8 @@ def self.parse_file(filename, name=nil)
   def self.parse(str, filename, name=nil)
     object = new
     object.name(name) if name.present?
-    ascii = str ? str.to_ascii : ''
-    object.instance_eval(ascii, filename)
+    content = str || ''
+    object.instance_eval(content, filename)
```

Comment:
nice find!

Reply:
Because it’s hard to revert a Rails database schema migration, we often keep code changes that use the new schema to a separate follow on Pull Request. This change seems pretty small, so probably not an issue, but if the new schema is compatible with the existing codebase, might be worth releasing this change later.

Reply:
Thanks, Suresh! Makes sense